e2e-tests/native/test/Action.ts

```diff
@@ -0,0 +1,74 @@
+type SwipeDirectionType = 'UP' | 'DOWN'
```

Comment:
Typescript supports enums and i would really recommend using them.

So this would change to:
enum SwipeDirectionType {
  Up,
  Down,
}
and us it later on swipte(SwipeDirectionType.Down) or is the switch/case. This allows auto-complete and  checks for wrong code.